examples/parallel/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+extern crate blas_src;
```

Comment:
Doesn't look like `blas_src` is used?

Reply:
No this is actually necessary as the resulting dylib otherwise does not link to BLAS. https://github.com/rust-ndarray/ndarray#how-to-enable-blas-integration has the details.

Reply:
Will add a comment with this link to the example.